tools/benchmark/src/Reporter.ts

```diff
@@ -120,21 +120,20 @@ export class BenchmarkReporter {
 		} else {
 			table.cell("status", `${pad(4)}${chalk.green("✔")}`);
 		}
+
 		table.cell("name", chalk.italic(testName));
-		table.cell("total time (s)", prettyNumber((result as BenchmarkData).elapsedSeconds, 2));
 
-		// Additional if-statement to display columns in more readble order.
+		// Additional if-statement to display columns in more readble order & avoid reporter from crashing if customData is empty.
```

Comment:
I think the comment can be removed at this point, the code around it has changed enough that it doesn't really add much to have it.